src/pages/home/report/ReportActionItem.tsx

```diff
@@ -658,6 +657,13 @@ function ReportActionItem({
             children = <IssueCardMessage action={action} />;
         } else if (ReportActionsUtils.isActionOfType(action, CONST.REPORT.ACTIONS.TYPE.EXPORTED_TO_INTEGRATION)) {
             children = <ExportIntegration action={action} />;
+        } else if (action.actionName === CONST.REPORT.ACTIONS.TYPE.RENAMED) {
```

Comment:
I think you can simplify this:

```suggestion
        } else if (ReportActionsUtils.isRenamedAction(action)) {
```

and that will make it simpler further down:

```ts
const initialMessage = ReportActionsUtils.getOriginalMessage(action);
```